### PR TITLE
add liquibase

### PR DESCRIPTION
**Pull request by acerlife**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,15 +8,129 @@
     <artifactId>SpringBootCalculator</artifactId>
     <version>1.0-SNAPSHOT</version>
 
- 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jooq</groupId>
+                <artifactId>jooq</artifactId>
+                <version>3.10.6</version>
```
Comment (VolodymyrMudryk) [inline pom.xml]: please extract to a property

File: pom.xml [hunk 2]
```diff
@@ -8,15 +8,129 @@
...
+            <dependency>
+                <groupId>org.jooq</groupId>
+                <artifactId>jooq-meta</artifactId>
+                <version>3.10.6</version>
+            </dependency>
+            <dependency>
+                <groupId>org.jooq</groupId>
+                <artifactId>jooq-codegen</artifactId>
+                <version>3.10.6</version>
+                <type>pom</type>
```
Comment (VolodymyrMudryk) [inline pom.xml]: why do you need this?

File: pom.xml [hunk 3]
```diff
@@ -8,15 +8,129 @@
...
         <plugins>
             <plugin>
                 <groupId>org.springframework.boot</groupId>
                 <artifactId>spring-boot-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <configuration>
+                    <source>1.8</source>
```
Comment (VolodymyrMudryk) [inline pom.xml]: please extract to a new property

File: pom.xml [hunk 4]
```diff
@@ -8,15 +8,129 @@
...
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-jdbc</artifactId>
+            <version>5.0.4.RELEASE</version>
+        </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>3.5.3</version>
```
Comment (VolodymyrMudryk) [inline pom.xml]: please extract to a new property

File: src/main/resources/liquibase/changelog.xml [hunk 5]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.1"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.1
+    http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.1.xsd">
+    <changeSet id="1" author="Yaroslav">
```
Comment (VolodymyrMudryk) [inline src/main/resources/liquibase/changelog.xml]: please use a small description in the ID instead

File: src/main/resources/liquibase/changelog.xml [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+        <createTable tableName="postfix_results">
+            <column name="id" type="int" autoIncrement="true">
```
Comment (VolodymyrMudryk) [inline src/main/resources/liquibase/changelog.xml]: should be `bigint` instead of `int`

File: src/main/resources/liquibase/changelog.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+                <constraints primaryKey="true" nullable="false"/>
+            </column>
+            <column name="string" type="varchar(50)">
```
Comment (VolodymyrMudryk) [inline src/main/resources/liquibase/changelog.xml]: what is `string` ? please use more clear name

File: pom.xml (hunk 1 above)
Comment (acerlife) [reply pom.xml]: is done

File: pom.xml (hunk 2 above)
Comment (acerlife) [reply pom.xml]: is deleted

File: pom.xml (hunk 3 above)
Comment (acerlife) [reply pom.xml]: is done

File: pom.xml (hunk 4 above)
Comment (acerlife) [reply pom.xml]: is done

File: src/main/resources/liquibase/changelog.xml (hunk 5 above)
Comment (acerlife) [reply src/main/resources/liquibase/changelog.xml]: is done

File: src/main/resources/liquibase/changelog.xml (hunk 6 above)
Comment (acerlife) [reply src/main/resources/liquibase/changelog.xml]: is done

File: src/main/resources/liquibase/changelog.xml (hunk 7 above)
Comment (acerlife) [reply src/main/resources/liquibase/changelog.xml]: is done

**Review by VolodymyrMudryk — APPROVED** [review]

**Pull request merged**